accepted/2020/workloads/workload-manifest.md

```diff
@@ -0,0 +1,430 @@
+
+## SDK Packs
+
+SDK packs are MSBuild SDK NuGet packages, but they are installed to `dotnet/packs/{pack-id}/{pack-version}`. They are expected to be automatically referenced by the targets.
+
+## Library Packs
+
+Library packs are normal NuGet packages. When a library pack is installed, the nupkg package file is placed in `dotnet/library-packs/` but are *not* extracted. This location is used as a directory feed by NuGet.
+
+If ones of these packages gets used, it will be extracted into the global packages folder. If the SDK is updated and the library pack is removed or replaced with a newer version, projects that have already been created and restored will continue to be able to use the extracted copy.
+
+This is intended to be used to pre-download NuGet packages referenced by templates so that a workload can work offline after installation. It should be used sparingly i.e. only for NuGets referenced by the core templates with default or popular options.
+
+## Template Packs
+
+Template packs are NuGet packages to be used by the dotnet templating engine. When a template pack is installed installed, the nupkg package file is placed in `dotnet/template-packs/` but is *not* extracted. Visual Studio and `dotnet new` are expected to automatically detect when templates nupkgs are installed and uninstalled to this folder and update their template hives automatically.
+
+
+## Tools Packs
+
+Tools packs are .NET global tools packages. Tools packs are installed to `dotnet/tools-packs/{package-id}/{package-version}/` and can be invoked via `dotnet` e.g. invoking `dotnet foo`, the runtime will use global.json and the manifest to determine the appropriate version of `dotnet-foo` to run.
+
+Tools packs are intended to be used for tools that are exposed part of the development experience. Tools that are only expected to be invoked by build targets should be distributed in an SDK pack.
+
+# Manifest Files
+
+## Composition
+
+Multiple manifests may be present in a .NET SDK band. Each manifest has its own ID and they are versioned independently. Components and packs may not be duplicated in multiple manifests within a band, though manifests may reference components and packs in other manifests that are expected to be installed.
+
+Manifests are not an arbitrary extensibility mechanism; they are intended to provide some flexibility in the distribution and composition of a coherent whole. For example, there could be a *dotnet* manifest used to update in-box shared runtimes, a *xamarin-ios* manifest used to describe and update the Xamarin.iOS optional components, and a *uwp* manifest used to describe and update the UWP optional components. These could all be produced from separate repositories using separate build and release processes.
```

Comment:
You almost get there, but it would be good to clarify the time dimension. They can be released at different times w/no coordination.